res/css/views/dialogs/security/_AccessSecretStorageDialog.scss

```diff
@@ -109,7 +109,7 @@ limitations under the License.
             text-align: initial;
             display: flex;
             flex-flow: column;
-            gap: 14px;
+            row-gap: 7px;
```

Comment:
It is based on: https://github.com/matrix-org/matrix-react-sdk/blob/859fdf7d51cb04c7926c370aee90b007bdf86d67/res/css/views/dialogs/security/_AccessSecretStorageDialog.scss#L21

I am not sure whether it would be fine to replace it with something like `$spacing-8`, so I would leave this as 7px.